imageregistry/v1/types.go

```diff
@@ -306,6 +306,38 @@ type ImageRegistryConfigStorageIBMCOS struct {
 	ServiceInstanceCRN string `json:"serviceInstanceCRN,omitempty"`
 }
 
+// ImageRegistryConfigStorageOSS holds Alibaba Cloud OSS configuration.
+// the registry to use Alibaba Cloud Object Storage Service for backend storage.
+// More about oss, you can look at the [official documentation](https://www.alibabacloud.com/help/product/31815.htm)
+type ImageRegistryConfigStorageOSS struct {
```

Comment:
Since there's another push coming anyway, let's clean this up to `ImageRegistryConfigStorageAlibaba`.  I think most of us won't recognize OSS

Reply:
Got it.